src/main/java/nl/Tech/It/Easy/Tech/It/Easy/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,75 @@
+package nl.Tech.It.Easy.Tech.It.Easy.controllers;
+import nl.Tech.It.Easy.Tech.It.Easy.exceptions.RecordNotFoundException;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.ArrayList;
+import java.util.List;
+@RestController
+@RequestMapping("/tvs")
+public class TelevisionsController {
+
+    private List<Televisions> tvs = new ArrayList<>();
+
+    public TelevisionsController() {
+        // Add some TV data to the list
+        tvs.add(new Televisions(55.5, 32, "4K Ultra HD", true, true, false, true, 5));
+        tvs.add(new Televisions(45.0, 40, "1080p Full HD", true, true, false, false, 2));
+        tvs.add(new Televisions(60.0, 50, "8K Ultra HD", true, true, true, true, 3));
+    }
+
+    // Get all TVs
+    @GetMapping("/tv")
```

Comment:
De "/tv" is hier niet nodig, omdat jou request mapping al "/tvs" zegt. Jou complete pad is nu namelijk "localhost:8080/tvs/tv"

Daarnaast is "/tv" enkelvoud, terwijl je hier de complete lijst returned. Het zou dus "/tvs" moeten zijn